### PR TITLE
Rshaw/openvpn 0.0.0 0.2

### PR DESCRIPTION
**Pull request by aggress**
Provides a new version of the openvpn package with the following top line changes since 0.0.0-0.1

1. Merges functionality from openvpn-admin and openvpn 0.0.0-0.1 packages
2. Adds security in the form of TLS & basic auth for the REST interface, znode access controls
3. Working synchronisation of openvpn assets to and from Zookeeper
4. Correct revocation of clients from openvpn and removal of assets

- TLS uses the self-signed certificate generated for OpenVPN itself
- Basic auth is configured through environment variables.

The src repository has now moved from https://github.com/mesosphere/dcos-openvpn to https://github.com/dcos-labs/dcos-openvpn which contains full instructions

Tested successfully via local universe

### CODE REVIEW

File: repo/packages/O/openvpn/1/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "2.0",
```
Comment (ryadav88) [inline repo/packages/O/openvpn/1/package.json]: please change `packagingVersion` to 4.0

File: repo/packages/O/openvpn/1/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "name": "openvpn",
+  "version": "0.0.0-0.2",
+  "scm": "https://github.com/dcos-labs/dcos-openvpn",
+  "description": "OpenVPN server and REST admin interface - community package",
+  "maintainer": "https://dcos.io/community/",
+  "tags": ["vpn", "remote-access"],
+  "preInstallNotes": "This release supercedes and merges both the openvpn and openvpn-admin 0.0.0-0.1 packages.\nPlease remove the openvpn-admin package if installed.\nNew features include basic auth, TLS and Zookeeper znode ACL.\n Please ensure you configure the ovpn_username and ovpn_password environment variables before deployment,\nas these are essential for access control to the REST interface and Zookeeper.\nPlease read the installation notes on https://github.com/dcos-labs/dcos-openvpn.",
```
Comment (ryadav88) [inline repo/packages/O/openvpn/1/package.json]: Add this line at the beginning of preInstallNotes: `Community packages are unverified and unreviewed content from the community`

File: repo/packages/O/openvpn/1/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (ryadav88) [inline repo/packages/O/openvpn/1/package.json]: please create a PR against dcos/examples repo to add an example documentation for this package. It is also a requirement now to submit a package to Universe. You can you the template from other examples there.

Comment (aggress) [reply repo/packages/O/openvpn/1/package.json]: https://github.com/dcos/examples/pull/214

File: repo/packages/O/openvpn/1/package.json (hunk 2 above)
Comment (aggress) [reply repo/packages/O/openvpn/1/package.json]: Done

File: repo/packages/O/openvpn/1/package.json (hunk 1 above)
Comment (aggress) [reply repo/packages/O/openvpn/1/package.json]: Done

**Pull request merged**